internal/utils/config/src/main/resources/ditto-mongo.conf

```diff
@@ -55,6 +55,10 @@ ditto.mongodb {
     useAwsIamRole = false
     useAwsIamRole = ${?USE_AWS_IAM_ROLE}
 
+    # Specifies the ARN of the AWS IAM Role to be assumed for MongoDB authentication.
+    awsRoleArn = ""
+    awsRoleArn = ${?AWS_ROLE_ARN}
```

Comment:
Adding a prefix `MONGO_DB_` would be good for the env var.
Also for other new variables.

Reply:
shall I add it also to USE_AWS_IAM_ROLE?